sdk/servicebus/Azure.Messaging.ServiceBus/api/Azure.Messaging.ServiceBus.netstandard2.0.cs

```diff
@@ -777,6 +780,31 @@ public sealed partial class TrueRuleFilter : Azure.Messaging.ServiceBus.Manageme
         public override string ToString() { throw null; }
     }
 }
+namespace Azure.Messaging.ServiceBus.Plugins
+{
+    public abstract partial class ServiceBusPlugin
+    {
+        protected ServiceBusPlugin() { }
+        public abstract string Name { get; }
```

Comment:
Who uses this?

Reply:
This can probably be removed. It was in Track 1 to prevent plugins from being registered multiple times, but I'm not sure why we would need that validation.